.github/workflows/auto-approve.yml

```
@@ -0,0 +1,26 @@
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  auto-approve:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+    - uses: actions/github-script@v6
```

actionlint (1.7.12)

[error] the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)